### PR TITLE
use pytest.approx to compare times

### PR DESCRIPTION
**Pull request by chipaca**

### CODE REVIEW

**Review by facundobatista — APPROVED** [review]
Thanks!

File: tests/commands/test_build.py [hunk 1]
```diff
@@ -612,9 +612,8 @@ def _test_build_generics_tree(tmp_path, caplog, *, expect_hardlinks):
             assert filecmp.cmp(str(p1), str(p2), shallow=False)
             assert p1.stat().st_mode == p2.stat().st_mode
             assert p1.stat().st_size == p2.stat().st_size
-            # checking st_atime seems to be a bit racy
-            # assert p1.stat().st_atime == p2.stat().st_atime
-            assert p1.stat().st_mtime == p2.stat().st_mtime
+            assert pytest.approx(p1.stat().st_atime, p2.stat().st_atime)
+            assert pytest.approx(p1.stat().st_mtime, p2.stat().st_mtime)
```
Comment (dstathis) [inline tests/commands/test_build.py:616]: I think the syntax here is not correct and will always pass. I believe you would need:

`p1.stat().st_mtime == pytest.approx(p2.stat().st_mtime)`

**Review by dstathis — CHANGES_REQUESTED** [review]

File: tests/commands/test_build.py (hunk 1 above)
Comment (facundobatista) [reply tests/commands/test_build.py:616]: You're right, Dylan!

**Review by dstathis — APPROVED** [review]

**Comment by chipaca** [comment]
@dstathis welcome back! thanks for the review

**Pull request merged**